doc/sphinx/language/core/inductive.rst

```diff
@@ -361,7 +361,7 @@ syntax: :n:`let fix @ident {* @binder } := @term in` stands for
 
 Some options of :n:`@fixannot` are only supported in specific constructs.  :n:`fix` and :n:`let fix`
 only support the :n:`struct` option, while :n:`wf` and :n:`measure` are only supported in
-commands such as :cmd:`Function` and :cmd:`Program Fixpoint`.
+commands such as :cmd:`Function` and :cmd:`Fixpoint`.
```

Comment:
Do not drop `Program` from here. If you want you can reformulate as "when the `program` attribute is used with `Fixpoint`".